tests/test_sdss.py

```diff
@@ -46,7 +46,7 @@ def testGetDR7(self):
         for ref in sub:
             im = ref.get("fpC")
             w, h = im.getWidth(), im.getHeight()
-            self.assertEqual(im.__class__, lsst.afw.image.ImageU)
+            self.assertEqual(im.__class__, lsst.afw.image.ExposureU)
```

Comment:
Does this actually pass?  It's confusing to have `ExposureF` above and then `ExposureU` here.  If this passes, then presumably the YAML should have `ExposureU`.

Reply:
It's definitely passing, and the SdssMapper.yaml is showing python lssst.afw.image.ExposureF under fpC.